src/pages/detail/movie/[id].vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <Nav>
+        <NavButton to="/overview/movies">
+            <!-- TODO: Extract icon to its own component  -->
```

Comment:
Maybe use @mdi/svg npm package?

Reply:
Ended up using `@mdi/js` and adding a custom Icon component, but this has been addressed.